modules/docs/src/04-codegen/01-customisation.md

```diff
@@ -134,3 +134,26 @@ whenever the `adtMember` trait is in use.
 Note: The `adtMember` trait has NO impact on the serialization/deserialization behaviors of smithy4s.
 The only thing it changes is what the generated code looks like. This is accomplished by keeping the
 rendered schemas equivalent, even if the case class is rendered in a different place.
+
+#### Specialized collection types
+
+Smithy supports `list` and `set`, Smithy4s renders that to `List[A]` and `Set[A]` respectively. You can also use the `@uniqueItems` annotation on `list` which is equivalent to `set`.
+
+Smithy4s has support for two specialized collection type: `Vector` and `IndexedSeq`. The following examples shows how to use them:
```

Comment:
```suggestion
Smithy4s has support for two specialized collection types: `Vector` and `IndexedSeq`. The following examples show how to use them:
```

Reply:
Addressed in [b3365de](https://github.com/disneystreaming/smithy4s/pull/303/commits/b3365de87c6af31653ec9810054c9d128c6a0e07)